source/usr/lib/mcvirt/parser.py

```diff
@@ -87,6 +87,11 @@ def __init__(self, verbose=True):
                                                        parents=[self.parent_parser])
         self.reset_parser.add_argument('vm_name', metavar='VM Name', type=str, help='Name of VM')
 
+
+        # Add arguments for fixing deadlock on a vm
+        self.deadlock_parser = self.subparsers.add_parser('clear_deadlock', help='Resolve the deadlock of a call to the daemon.',
```

Comment:
The rest of the arguments use dashes, so would probably prefer 'clear-deadlock'.
The term 'deadlock' has not been used anywhere in MCVirt. Would probably prefer something like 'lock clear-method-lock'